docs/product_guides/product_guides.md

```diff
@@ -0,0 +1,22 @@
+---
+description: Dicover various of Ibexa DXP features.
+page_type: landing_page
+---
+
+# Product guides
```

Comment:
I would like to discuss the nomenclature with the rest of the team. We have guides that are rather "feature guides" than product guides. Only the ones that refer to product editions are product guides. Could we start calling them simply "guides" or "feature guides"?
@ibexa/documentation WDYT?